### PR TITLE
add Js Block unit tests

### PR DESCRIPTION
**Pull request by danac-gs**
I merged develop into this branch. We could keep it after merging back.

### CODE REVIEW

**Comment by YevhenShBolt** [comment]
I somewhere saw the article where we can use getMockBuilder instead of invokeMethod to call the private method... but now i cannot find it, if i find it i will share with team.

File: Test/Unit/Block/JsTest.php [hunk 1]
```diff
@@ -241,4 +247,55 @@ public function setSandboxMode($value = true)
             ->method('isSandboxModeSet')
             ->will($this->returnValue($value));
     }
+
+    /**
+     * @inheritdoc
+     */
+    public function testGetJavascriptSuccess()
```
Comment (daisy1754) [inline Test/Unit/Block/JsTest.php]: You aren't testing anything meaningful here :)
This test is testing mock framework, not our code.
I suggest removing this.

Comment (danac-gs) [reply Test/Unit/Block/JsTest.php]: removed

File: Test/Unit/Block/JsTest.php [hunk 2]
```diff
@@ -241,4 +247,39 @@ public function setSandboxMode($value = true)
...
+        $method = $reflection->getMethod($methodName);
+        $method->setAccessible(true);
+
+        return $method->invokeArgs($object, $parameters);
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function testGetQuoteIsVirtual()
```
Comment (daisy1754) [inline Test/Unit/Block/JsTest.php]: again, I think this test doesn't test anything interesting. tested method is basically one-line getter so having test doesn't provide much value here.

**Pull request closed without merging**